res/service-account.yml

```diff
@@ -62,3 +62,39 @@ roleRef:
   apiGroup: rbac.authorization.k8s.io
   kind: ClusterRole
   name: netobserv-cli
+---
+kind: ClusterRole
+apiVersion: rbac.authorization.k8s.io/v1
+metadata:
+  name: netobserv-cli-metrics
+  namespace: "{{NAMESPACE}}"
+rules:
+# allow openshift-monitoring metrics reading
+  - apiGroups:
+     - ''
+    resources:
+     - pods
+     - services
+     - endpoints
+    verbs:
+     - list
+     - get
+     - watch
+  - nonResourceURLs:
+     - /metrics
+    verbs:
+     - get
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRoleBinding
+metadata:
+  name: netobserv-cli-metrics
+  namespace: "{{NAMESPACE}}"
```

Comment:
not a blocker, but it doesn't make sense to set `namespace` on a clusterrolebinding (same for clusterrole) - it will work but will be ignored afaik

Reply:
Ultimately I would like to rely on Role / RoleBinding here but it doesn't seems to work

Let's keep as is for this release and we'll investigate